backend/services/few_shot_examples.py

```
@@ -0,0 +1,228 @@
+        base_examples = self.get_role_specific_examples(persona_data)
+        
+        if attempt_number > 3:
+            # More helpful examples for struggling users
+            adaptive_guidance = f"""
+
+ADAPTIVE GUIDANCE (User has tried multiple times):
+- Be more direct and helpful in your responses
+- Ask clarifying questions to understand what they need
+- Provide more specific insights and suggestions
+- Guide them toward the right direction more explicitly
+"""
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)